services/viva-ms/lambdas/generateRecurringCaseHtml.js

```diff
@@ -174,22 +127,21 @@ export async function main(event, context) {
   return true;
 }
 
-function updateVivaCaseState(caseItem) {
-  const updateParams = {
-    TableName: config.cases.tableName,
-    Key: {
-      PK: caseItem.PK,
-      SK: caseItem.SK,
-    },
-    UpdateExpression: 'SET #state = :newState',
-    ExpressionAttributeNames: {
-      '#state': 'state',
-    },
-    ExpressionAttributeValues: {
-      ':newState': CASE_HTML_GENERATED,
-    },
-    ReturnValues: 'NONE',
-  };
-
-  return dynamoDb.call('update', updateParams);
+function getChangedCaseAnswerValues(currentAnswerList, previousAnswerList) {
+  return currentAnswerList.map(answer => {
+    const tags = [...answer.field.tags];
+
+    const previousAnswer = previousAnswerList.find(item => item.field.id === answer.field.id);
```

Comment:
`item` might be renamed to `previousAnswer`?
Then `answer.field.id` would become `currentAnswer.field.id`